R/plot_functions.R

```diff
@@ -969,53 +972,65 @@ plot_mod.mixmod_em <- function(p_obj, x, title_trace = "Fit", ...) {
 #'               112, 108, 104, 99, 99, 96, 94)
 #' status <- c(rep(0, 5), rep(1, 67))
 #'
-#' data <- reliability_data(x = cycles, status = status)
-#'
-#' tbl_john <- estimate_cdf(data, methods = c("johnson", "nelson"))
+#' tbl_john <- estimate_cdf(x = cycles, status = status, method = "johnson")
 #'
 #' # Example 1: Probability Plot and Regression Line Three-Parameter-Weibull:
 #' plot_weibull <- plot_prob(
-#'   tbl_john,
+#'   x = tbl_john$x,
+#'   y = tbl_john$prob,
+#'   status = tbl_john$status,
+#'   id = tbl_john$id,
 #'   distribution = "weibull",
 #'   title_main = "Three-Parametric Weibull",
 #'   title_x = "Cycles",
 #'   title_y = "Probability of Failure in %",
 #'   title_trace = "Failed Items"
 #' )
 #'
-#' mrr <- rank_regression(
-#'   tbl_john,
+#' rr <- rank_regression(
+#'   x = tbl_john$x,
+#'   y = tbl_john$prob,
+#'   status = tbl_john$status,
 #'   distribution = "weibull3",
-#'   conf_level = .90
+#'   conf_level = 0.9
```

Comment:
0.90 ?

Reply:
I thought we agreed on using leading 0 and removing trailing 0